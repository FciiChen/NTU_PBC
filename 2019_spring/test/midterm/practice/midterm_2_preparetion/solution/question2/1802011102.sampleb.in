John, Male, 30, Taiwan
Mary, Female, 40, USA
BREAK
MARY, E23531
John, Q23331
BREAK