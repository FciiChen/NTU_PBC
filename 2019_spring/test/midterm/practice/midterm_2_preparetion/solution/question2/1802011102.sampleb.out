John       Male       30         Taiwan     Q23331    
Mary       Female     40         USA        E23531    
